frontend/src/screens/learner.tsx

```diff
@@ -114,15 +114,67 @@ export const SearchResults: FC<{search: string, filteredStudies: ParticipantStud
     )
 }
 
+const Circle = styled.div({
+    borderRadius: '50%',
+    border: `1px solid ${colors.blue}`,
+    backgroundColor: colors.white,
+    width: '.875rem',
+    height: '.875rem',
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+})
+
+const StudyDuration: FC<{duration: Set<Number>, setDuration: Function, durationText: Number}> = ({ duration, setDuration, durationText }) => {
+
+    const handleDurationChange = (duration:Number) => {
+        setDuration((prev:Set<Number>) => {
+            const newDuration = new Set<Number>(prev)
+            if(newDuration.has(duration)){
+                newDuration.delete(duration)
+            }else{
+                newDuration.add(duration)
+            }
+            return newDuration
+        })
+    }
+
+    const [active, setActive] = useState<Boolean>(() => {
+        return duration.has(durationText)
+    })
+
+    useEffect(() => {
+        setActive(() => {
+            return duration.has(durationText)
+        })
+    }, [duration, durationText])
+
+    return (
+        <Flex justify='center' align='center' gap='.5rem' 
+            pt='.25rem' pb='.25rem' pl='.625rem' pr='.625rem'
+            bg={ active? colors.blue: colors.white }
+            style={{ border: `1px solid ${colors.blue}`, borderRadius: '50rem', transition: 'all .1s ease-in' }} 
+            onClick={() => {
+                handleDurationChange(durationText)
+            }}>
+            <Text size='sm' c={ active? colors.white: colors.blue }>~{String(durationText)} min</Text> 
+            <Circle><Icon height={10} color={colors.blue} icon={active? 'minus': 'plus'}></Icon></Circle> 
```

Comment:
For Icons, i think we want to try and move towards using `@tabler/icons-react`, trying ton consolidate and eventually remove uses of this `<Icon>` component here - let me know if that makes sense